themes/default/css/_caColorbox.scss

```diff
@@ -1,20 +1,14 @@
-//_caColorbox.scss
```

Comment:
Most of the *.scss files were changed only for whitespace and these (inconsistently applied and useless) comments.  It was driving me nuts so I fixed it.